src/otx/recipe/_base_/data/keypoint_detection.yaml

```diff
@@ -6,20 +6,18 @@ data_format: coco_person_keypoints
 unannotated_items_ratio: 0.0
 image_color_channel: RGB
 input_size:
-  - 256
-  - 192
+  - 512
+  - 512
 train_subset:
   subset_name: train
   batch_size: 32
+  num_workers: 2
```

Comment:
Is there a problem when num workers is not 2?

Reply:
No problem, I just aligned it with other tasks and added the explicit value in config